readme.md

```diff
@@ -56,7 +56,7 @@ recommended way of ensuring this is using a crontab like this:
 
 ```
 04	00	*	*	*	php www/index.php notifications:assignment-deadlines "1 day"
-02	00	*	*	*	php www/index.php uploads:cleanup
+02	00	*	*	*	php www/index.php db:cleanup:uploads
```

Comment:
You could probably add the other commands here too.

Reply:
Well... it depends if we do want to do this regularly or not. In my opinion cleanups of configs and localized texts might by done once or twice a year, not every day or week...